<?xml version="1.0" encoding="utf-8"?>

<launch>
	<node pkg="object_detection_openvino" type="object_detection_openvino" name="object_detection" output="screen">
		<param name="info_topic"				value="/webcam/camera_info"/>
		<param name="color_topic"				value="/webcam/image_raw"/>
		<param name="detection_image_topic"		value="image_raw"/>
		<param name="detection_info_topic"		value="detection_info"/>
		<param name="detection_2d_topic"		value="detections_2d"/>
		<param name="model_thresh"				value="0.4"/>
		<param name="model_iou_thresh"			value="0.4"/>
		<param name="model_xml"					value="$(find object_detection_openvino)/models/mobilenet-ssd/FP16/mobilenet-ssd.xml"/>
		<param name="model_bin"					value="$(find object_detection_openvino)/models/mobilenet-ssd/FP16/mobilenet-ssd.bin"/>
		<param name="model_labels"				value="$(find object_detection_openvino)/models/mobilenet-ssd/FP16/mobilenet-ssd.labels"/>
		<param name="model_type"				value="SSD"/>
		<param name="device_target"				value="CPU"/>
		<param name="show_fps"					value="false"/>
		<param name="output_image"				value="true"/>
	</node>
</launch>

